package.json

```diff
@@ -12,13 +12,15 @@
     "test": "echo \"No test specified\" && exit 0"
   },
   "dependencies": {
+    "@capacitor/core": "^2.0.1",
     "@quasar/extras": "^1.0.0",
     "axios": "^0.18.1",
     "gitmoji-cli": "^3.2.3",
     "npm": "^6.14.4",
     "quasar": "^1.0.0",
     "vue-i18n": "^8.16.0",
-    "vuex-persistedstate": "^3.0.1"
+    "vuex-persistedstate": "^3.0.1",
+    "webpack": "^4.42.1"
```

Comment:
Oh what's that?
I think webpack is not needed.

Reply:
Ok, I have no clue why this was there.